### PR TITLE
Ixia support for version field

### PR DESCRIPTION
**Pull request by anjan-keysight**
@marcushines please pull the ixia node related changes for version support

### CODE REVIEW

**Comment by anjan-keysight** [comment]
@marcushines please pull the changes required for version support for Ixia nodes.

**Review by alexmasi — DISMISSED** [review]

File: topo/node/node.go [hunk 1]
```diff
@@ -512,8 +514,13 @@ var (
 func GetNextPort() uint32 {
 	muPort.Lock()
 	defer muPort.Unlock()
-	p := nextPort
-	nextPort++
+	ok := true
```
Comment (marcushines) [inline topo/node/node.go:517]: why is this being added as part of this pull Request?

File: topo/topo.go [hunk 2]
```diff
@@ -124,6 +124,18 @@ func New(kubecfg string, tpb *topopb.Topology, opts ...Option) (*Manager, error)
 
 // Load creates an instance of the managed topology.
 func (m *Manager) Load(ctx context.Context) error {
+	node.UsedNodePortMap = make(map[int32]bool)
```
Comment (marcushines) [inline topo/topo.go:127]: what is the goal here?

File: topo/node/node.go (hunk 1 above)
Comment (anjan-keysight) [reply topo/node/node.go]: I have reverted those changes. Those changes was made for another issue and it got accidentally pushed with the PR still open.

File: topo/topo.go (hunk 2 above)
Comment (anjan-keysight) [reply topo/topo.go]: @marcushines same response as above.

**Comment by marcushines** [comment]
no worries that was my fault for not getting this merged sooner

**Review by marcushines — APPROVED** [review]

**Pull request merged**